packages/statemanager/test/statelessVerkleStateManager.spec.ts

```diff
@@ -43,4 +52,115 @@ describe('StatelessVerkleStateManager: Kaustinen Verkle Block', () => {
       'should have correct codeHash'
     )
   })
+
+  it('put/delete/modify account', async () => {
+    const stateManager = new StatelessVerkleStateManager({ common })
+    stateManager.initVerkleExecutionWitness(block.executionWitness)
+
+    const address = new Address(randomBytes(20))
+
+    let check = await stateManager.getAccount(address)
+    assert.isUndefined(check, 'should return undefined for nonexistent account')
```

Comment:
In our current implementation, we are throwing whenever the account is not found in the statelessStateManager (i.e. if it's not in the cache, and not in the witness). An "non-existent" account would be specified as `null` (or `0x`) in the witness.